src/pkg/packager/lint/lint.go

```diff
@@ -24,22 +31,29 @@ func getSchemaFile() ([]byte, error) {
 	return ZarfSchema.ReadFile("zarf.schema.json")
 }
 
-// ValidateZarfSchema validates a zarf file against the zarf schema, returns *validator with warnings or errors if they exist
+// ValidateZarf validates a zarf file against the zarf schema, returns *validator with warnings or errors if they exist
 // along with an error if the validation itself failed
-func ValidateZarfSchema(path string) (*Validator, error) {
+func ValidateZarf(createOpts types.ZarfCreateOptions) (*Validator, error) {
```

Comment:
```suggestion
func Validate(createOpts types.ZarfCreateOptions) (*Validator, error) {
```

Could just be simplified to Validate